console-subscriber/src/builder.rs

```diff
@@ -268,6 +274,14 @@ impl Builder {
         Self { self_trace, ..self }
     }
 
+    #[cfg(feature = "grpc-web")]
+    pub fn enable_grpc_web(self, enable_grpc_web: bool) -> Self {
```

Comment:
Please add some documentation here. A brief description of what this option enables and also an explanation that  `serve_with_grpc_web` can be used to provide more advanced configuration options (including a link to the docs for that method).

Reply:
Added.